redirect.py

```diff
@@ -41,8 +41,20 @@ def get(self):
           self.response.write(json.dumps(result))
       elif self.request.get('site') == 'comments':
           urlfetch.set_default_fetch_deadline(10)
+          startIndex = self.request.get('start')
+
+          try:
+            startIndex = int(startIndex)
+          except:
+            startIndex = 0
+
+          maxResults = self.request.get('max')
+          try:
+            maxResults = int(maxResults)
+          except:
+            maxResults = 0
           self.response.headers.add_header("Access-Control-Allow-Origin", "*")
-          result = monorail.issues().comments().list(projectId='chromium', issueId=self.request.get('issueId')).execute()
+          result = monorail.issues().comments().list(projectId='chromium', issueId=self.request.get('issueId'), maxResults=max, startIndex=startIndex).execute()
```

Comment:
I found a few sample CLs.
>100 comments - http://crbug.com/668012
>200 comments - http://crbug.com/628043
I tried to find an issue with more than 500 comments and failed miserably. 

This is the script I used:

```
var CHROMEZ_COMMENTS = 'http://chromez-app.appspot.com/redirect/?site=comments&issueId='

function myFunction() {
  for (var i=start; i<end; i++) {
    var current_query = CHROMEZ_COMMENTS + encodeURIComponent(i);
    var fetch = UrlFetchApp.fetch(current_query);
    var response = JSON.parse(fetch)
    if (response.totalResults > 400) {
      Logger.log("huzzah " + i)
      break;
    }
  }
}
```

Fair warning I only used it for issues 50000-60000. :P

Also, I don't know how to test the number of comments I can get locally - with my change if I redirect to http://localhost:8080/redirect/?site=comments&issueId=668012 - I get a 500 error